app/src/main/java/com/globant/practice/presentation/view/adapter/SubscriberAdapter.java

```diff
@@ -0,0 +1,132 @@
+package com.globant.practice.presentation.view.adapter;
+
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+import com.globant.practice.R;
+import com.globant.practice.domain.model.User;
+import java.util.List;
+
+/**
+ * Adapter of the RecyclerView on HomeActivity
+ * Created by jonathan.vargas on 17/04/2017.
+ */
+public class SubscriberAdapter extends RecyclerView.Adapter<SubscriberAdapter.SubscriberViewHolder> {
+
+    /**
+     * Provides the manager of the user clicks on the RecyclerView
+     */
+    public interface OnUserClickListener {
+
+        /**
+         * Manages the user clicks
+         *
+         * @param user User instance of the object that the user makes click
+         */
+        void onUserClick(User user);
+    }
+
+    private List<User> users;
+    private OnUserClickListener onUserClickListener;
+
+    /**
+     * Construct method of the SubscriberAdapter
+     *
+     * @param onUserClickListener OnUserClickListener instance
+     */
+    public SubscriberAdapter(OnUserClickListener onUserClickListener) {
+        this.onUserClickListener = onUserClickListener;
+    }
+
+    /**
+     * Set the values of the user list
+     *
+     * @param users user list
+     */
+    public void setAdapterData(List<User> users) {
+        this.users = users;
```

Comment:
After setting the items, you are not refreshing the adapter so the items will not be displayed.